mapper/orm/src/main/java/org/hibernate/search/mapper/orm/loading/impl/HibernateOrmMassIndexingOptions.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.orm.loading.impl;
+
+import org.hibernate.CacheMode;
+import org.hibernate.search.mapper.pojo.massindexing.loader.MassIndexingOptions;
+
+public interface HibernateOrmMassIndexingOptions extends MassIndexingOptions {
```

Comment:
It's not necessary to extend a common superinterface `MassIndexingOptions` here and in the JavaBean module. We generally try to avoid common APIs in the two mappers (even though you already did that elsewhere...).

Please rename this interface to `MassIndexingOptions`, and move all relevant methods from `org.hibernate.search.mapper.pojo.massindexing.loader.MassIndexingOptions` to here.